Analysis/include/AnalysisCategories.h

```diff
@@ -233,6 +236,14 @@ struct EventCategory {
         return *btag_wp;
     }
 
+    bool HasStrictBtagConstraint() const { return strict_n_btag.is_initialized(); }
```

Comment:
if btag constraint is specified that strict is specified as well. So, don't need HasStrictBtagConstraint method.